client/package.json

```diff
@@ -198,6 +198,12 @@
             "type": "number",
             "default": 1,
             "description": "Amount of workers assigned to proofs in delegation mode"
+          },
+          "vscoq.proof.block": {
+              "scope": "window",
+              "type": "boolean",
+              "default": true,
```

Comment:
Should I change it to false by default maybe ?

Reply:
I really don't know. Probably the new default is less surprising.

One thing I'd like to change is that, even if it is off, going down 1 sentence blocks. But I got used to that from the CoqIDE days, so I don't know it it is just me. I mean: if I say go there, and there is far away, I really want to go there. If I want to execute the next sentence, and if it fails, I want to stay where I am. So to me ALT-down should have the ~should_block_on_error:true always, while ALT-right should read the default from the settings.

Reply:
I thought a bit more about my use case, and maybe also the one of Laurent. I think moving the caret in case of a failing step forward is bad, it really seems step fwd is special.

Reply:
Okay ! Just changed it

Reply:
So if I understand correctly, the caret should not move at all on step forward ?